datahub-web-react/src/conf/theme/types.ts

```diff
@@ -16,6 +16,8 @@ export type Theme = {
         'steps-nav-arrow-color': string;
         'homepage-background-upper-fade': string;
         'homepage-background-lower-fade': string;
+        'box-shadow': string;
+        'box-shadow-hover': string;
     };
```

Comment:
looks like antd has some shadow variables already:

https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less#L154

would it make sense to use one of those instead? haven't looked into it too much.